.lando.yml

```diff
@@ -5,10 +5,10 @@ env_file:
 services:
   core:
     type: node:custom
-    command: npx nodemon -w packages -w apis --verbose --ext ts,ttl,js,trig --ignore **/*.test.ts --ignore **/*.spec-graphs.ts --exec node --inspect=0.0.0.0:45671 -r ts-node/register --inspect apis/core/index.ts
+    command: mkdir -p /var/www/.npm-global/lib && npx nodemon -w packages -w apis --verbose --ext ts,ttl,js,trig --ignore **/*.test.ts --ignore **/*.spec-graphs.ts --exec node --inspect=0.0.0.0:45671 -r ts-node/register --inspect apis/core/index.ts
```

Comment:
What's in the `/var/www/.npm-global/lib` dir?

Reply:
Calling `npx nodemon` will install it first, and to install it, this path is needed.
It seems that in the node 18 container this directory is missing and that it is created on the fly, that's why it needed to create it manually.